bundles/org.openhab.binding.airq/src/main/java/org/openhab/binding/airq/internal/AirqHandler.java

```diff
@@ -0,0 +1,797 @@
+                                channelUID.getId(), command.toString());
+                    }
+                    break;
+                case "averagingRhythm":
+                    try {
+                        newobj.addProperty("SecondsMeasurementDelay", Integer.parseUnsignedInt(command.toString()));
+                    } catch (Exception exc) {
+                        logger.warn(
+                                "air-Q - airqHandler - handleCommand(): {} only accepts an integer value, and {} is not.",
+                                channelUID.getId(), command.toString());
+                    }
+                    break;
+                case "powerFreqSuppression":
+                    String newFreq = command.toString();
+                    if (newFreq.equals("50Hz") || newFreq.equals("60Hz") || newFreq.equals("50Hz+60Hz")) {
+                        newobj.addProperty("Rejection", newFreq);
+                        changeSettings(newobj);
+                    } else {
+                        logger.warn(
+                                "air-Q - airqHandler - handleCommand(): {} should be set to {} but it isn't a correct setting for the power frequency suppression (only 50Hz or 60Hz)",
+                                channelUID.getId(), command.toString());
+                    }
+                    break;
+                default:
+                    logger.warn(
+                            "air-Q - airqHandler - handleCommand(): unknown command {} received (channelUID={}, value={})",
+                            command, channelUID, command);
+            }
+        }
+    }
+
+    @Override
+    public void initialize() {
+        config = getThing().getConfiguration().as(AirqConfiguration.class);
+        logger.debug("air-Q - airqHandler - initialize(): config={}", config);
+        updateStatus(ThingStatus.UNKNOWN);
+        // We don't have to test if ipAddress and password have been set because we have defined them
+        // as being 'required' in thing-types.xml and OpenHAB will only initialize the handler if both are set.
+        String data = getDecryptedContentString("http://".concat(config.ipAddress.concat("/data")), "GET", null);
+        // we try if the device is reachable and the password is correct. Otherwise a corresponding message is
+        // thrown in Thing manager.
+        if (data == null) {
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR,
+                    "Unable to retrieve get data from air-Q device. Probable cause: invalid password.");
+        } else {
+            updateStatus(ThingStatus.ONLINE);
+        }
+        pollingJob = scheduler.scheduleWithFixedDelay(this::pollData, 0, POLLING_PERIOD_DATA, TimeUnit.MILLISECONDS);
+        getConfigDataJob = scheduler.scheduleWithFixedDelay(this::getConfigData, 0, POLLING_PERIOD_CONFIG,
+                TimeUnit.MINUTES);
+        logger.debug("air-Q - airqHandler - initialize() finished");
```

Comment:
This should be removed, see the dev guidelines on logging as it shows this as an example.